lib/rust/ensogl/component/shadow/src/lib.rs

```diff
@@ -46,6 +46,8 @@ pub struct Parameters {
 
 /// Loads shadow parameters from the given style, at the given path. The structure of the style
 /// definition should be analogous to that at `ensogl_hardcoded_theme::shadow`.
+#[allow(clippy::needless_borrow)] // This produces false positives that do not recognise that the
```

Comment:
Can we post a link to a report about the false positive please, so we will remove it after its fixed?

Reply:
Turns out I missed something here and the borrow was needles after all. So, the `allow` is now removed (here and below).